code/game/dna/genes/goon_disabilities.dm

```diff
@@ -37,9 +37,9 @@
 		block=RADBLOCK
 
 	OnMobLife(var/mob/living/owner)
-		owner.apply_effect(max(owner.radiation, 20), IRRADIATE)
+		owner.apply_effect(max(owner.radiation, 5), IRRADIATE)
```

Comment:
this max() shouldn't exist and is probably why you are getting the "10 second death sentence" experience
this literally reads
Pick the bigger of 2 values, owner.radiation, or 5.
So say, the owner has no radiation. 0 < 5, apply 5 radiation.
Now owner has 5 radiation. 5 == 5, apply 5 radiation.
Now owner has 10 radiation. 10 > 5, apply 10 radiation.
Now owner has 20 radiation. 20 > 5, apply 20 radiation.
you get the idea